core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/EmbeddedTextEditorPresenter.java

```diff
@@ -197,11 +198,17 @@ public void onReady(final String content) {
             @Override
             public void onError() {
                 displayErrorPanel(constant.editorInitErrorMessage());
+                if (callback != null) {
```

Comment:
I think yes, because for example in class  RecipeEditorPanel we creates simple editor panel but we don't need OpenEditorCallback, or we can use empty implementation. What do you think?